media-source/MediaSource-addSourceBuffer_null.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>MediaSource#addSourceBuffer(null) and MediaSource#addSourceBuffer("")</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+
+<script>
+async_test(function(t) {
```

Comment:
use media_test(...) or something under it, to auto-generate a <body> in the document.